LoveMachine.Core/ButtplugController.cs

```diff
@@ -192,6 +220,23 @@ protected void DoVibrate(float intensity, int girlIndex, Bone bone)
             client.VibrateCmd(intensity, girlIndex, bone);
         }
 
+        protected internal IEnumerator DoRotate(int girlIndex, Bone bone,
+                    float strokeTimeSecs)

```

Comment:
You should probably include clockwise as a parameter here. Then you can demote both random and clockwise to local variables in RunRotatorLoop.

Reply:
Yeah, I did such way but I noticed we need to reimplement `clockwise` handling in `TestRotator` as well.
I this it's better to encapsulate this logic in DoRotate since currently it's just changed randomlly.

```
if (random.NextDouble() <= CoreConfig.RotationDirectionChangeChance.Value)
            {
                clockwise = !clockwise;
            }
```

What do you think?

Reply:
I don't really get why randomness is needed for testing. Why not just alternate back and forth?

Reply:
> why randomness is needed for testing

I think it's better to adjust `CoreConfig.RotationDirectionChangeChance.Value` via test button though, if you think it isn't needed, I'll apply changes

Reply:
You don't have to worry about that, because device testing is just an ugly hack at this point, even for strokers. So I'll have to fix that eventually anyway.